Package.swift

```diff
@@ -14,7 +18,11 @@ let package = Package(
                  targets: ["Optimizely"])
     ],
     targets: [
-        .target(name: "Optimizely", path: "Sources")
+        .target(
+            name: "Optimizely",
+            path: "Sources",
+            resources: [.copy("Supporting Files/PrivacyInfo.xcprivacy")]
```

Comment:
I think you need to exclude the file and add it in the resource. I couldn't find documentation saying it but I checked many libraries that are doing that.

Reply:
> I think you need to exclude the file and add it in the resource. I couldn't find documentation saying it but I checked many libraries that are doing that.

I also haven't found any proper documentation so far. But I think exclude and added as resource same file is a kind of self-contradictory to me. I also checked it works well and didn't throw any warning or error.

Reply:
I think the reasoning is to not add it in the source code but only inside the resource bundle. But if both works then choose whatever makes more sense for you.